TapOrHoldCounter/MainVC.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  MainVC.swift
+//  TapOrHoldCounter
+//
+//  Created by vmalikov on 1/28/16.
+//  Copyright © 2016 justForFun. All rights reserved.
+//
+
+import UIKit
+
+class MainVC: UIViewController {
```

Comment:
@AliceAponasko I chose `MainVC` because it's short enough and make sense, isn't?